Makefile

```diff
@@ -165,8 +166,13 @@ build-benchmark-docker: ## Build benchmark test docker image for NGINX Plus, nee
 		-f test/docker/Dockerfile .
 
 # Install tests
-test-install: ## Run agent install test
-	GOWORK=off CGO_ENABLED=0 go test -v ./test/install
+test-install: ## Run agent install/uninstall test for release
```

Comment:
this should probably roll up into an integration target, with multiple tests suites running against that and a new CI/CD stage